pkg/account/user.go

```diff
@@ -56,13 +56,15 @@ func (s *server) CurrentUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	webhandlers.JSON(w, r, struct {
-		User       json.RawMessage `json:"user"`
-		LoggedInAt *time.Time      `json:"logged_in_at"`
-		SessionId  string          `json:"session_id"`
+		User             json.RawMessage `json:"user"`
+		LoggedInAt       *time.Time      `json:"logged_in_at"`
+		SessionId        string          `json:"session_id"`
+		SessionExpiresAt *time.Time      `json:"expires_at"`
 	}{
-		User:       userJSON,
-		LoggedInAt: ttnpb.StdTime(session.CreatedAt),
-		SessionId:  session.SessionId,
+		User:             userJSON,
+		LoggedInAt:       ttnpb.StdTime(session.CreatedAt),
+		SessionExpiresAt: ttnpb.StdTime(session.ExpiresAt),
+		SessionId:        session.SessionId,
```

Comment:
Nit: Prefer to keep the original ordering of the struct fields.

```suggestion
		SessionId:        session.SessionId,
		SessionExpiresAt: ttnpb.StdTime(session.ExpiresAt),
```